### PR TITLE
Waralee  Documentation Update

### PR DESCRIPTION
**Pull request by bobgeorgethe3rd**
Mostly re-formats to bring up to date.
 
The pocket section is now ready for #1575 as well. Originally suppose to be same update but I needed to bring this part up sooner as my workload is getting quite big so have left the section and added a Fixme instead.

### CODE REVIEW

File: markdown/org/docs/patterns/waralee/cutting/en.md [hunk 1]
```diff
@@ -1,18 +1,37 @@
-Waralee consists of two identical pieces of fabric.
+<Note>
+
+Waralee pants parts consists of two mirrored pieces of fabric.
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/cutting/en.md:3]: "Waralee pants parts consist" or "Waralee pants consist" or "Waralee consists"?

File: markdown/org/docs/patterns/waralee/cutting/en.md [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
 They are basically big rectangles with pieces missing.
-As such, it is easier to print out the piece you need to cut out and measure the rest,
+As such, it is easier to print out the part you need to cut out and measure the rest,
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/cutting/en.md:5]: I guess the idea is for this sentence to agree with the earlier one with regard to the usage of "piece" or "part".

File: markdown/org/docs/patterns/waralee/cutting/en.md [hunk 3]
```diff
@@ -1,18 +1,37 @@
...
 than it is to print out everything and cut the pattern the traditional way.
 
-The pattern is available both ways.
-You can print out the whole thing, tape all the pages together, and use that to cut out the fabric.
-It is also possible to print out only the part that you need to cut out.
-This part has a negative seam allowance, since you'll be using it opposite of what you normally use the pattern pieces for.
+However the pattern is available both ways so you can print out the whole thing, tape all the pages together, and use that to cut out the fabric if you prefer but we recommend only printing the part that you need to cut out.
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/cutting/en.md:8]: This sentence felt a little long to me and I had to reread it. A small suggestion:
```suggestion
That said, the pattern is available both ways. So you can print out the whole thing, tape all the pages together, and use that to cut out the fabric, if you prefer. But we recommend only printing the part that you need to cut out.
```

File: markdown/org/docs/patterns/waralee/cutting/en.md [hunk 4]
```diff
@@ -1,18 +1,37 @@
...
+
+You will need to print out the pocket and pocket facings in the traditional way.
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/cutting/en.md:10]: Maybe "pocket and pocket facing" or "pockets and pocket facings"?

File: markdown/org/docs/patterns/waralee/cutting/en.md (hunk 1 above)
Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/waralee/cutting/en.md]: Part when pattern, peice when fabric I believe.

File: markdown/org/docs/patterns/waralee/cutting/en.md (hunk 2 above)
Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/waralee/cutting/en.md:5]: part for pattern

File: markdown/org/docs/patterns/waralee/cutting/en.md (hunk 3 above)
Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/waralee/cutting/en.md:8]: yea this better paced. I mostly just did basica reformat not wording work on this updated

File: markdown/org/docs/patterns/waralee/cutting/en.md (hunk 4 above)
Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/waralee/cutting/en.md]: plural, typo

File: markdown/org/docs/patterns/waralee/cutting/en.md [hunk 5]
```diff
@@ -1,18 +1,37 @@
...
+ - **Main fabric**
+   - Cut the **2 pants** parts.
+   - (Optional) Cut **2 Pocket** parts on the fold.
+   - Cut **4 backPocket** parts
+   - Cut **4 frontFacing** parts
+   - Cut **4 backFacing** parts
+
+<Note>
+
+You can cut the Pocket and backPocket out of a lining fabric if you prefer but this with only be needed if you deviate from the recommended main fabrics.
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/cutting/en.md:24]: Small typo and a suggestion for making the number of main fabrics (1 I think) clearer:
```suggestion
You can cut the Pocket and backPocket out of a lining fabric if you prefer but this will only be needed if you deviate from the main fabric recommendations.
```

File: markdown/org/docs/patterns/waralee/cutting/en.md [hunk 6]
```diff
@@ -1,18 +1,37 @@
...
-This part has a negative seam allowance, since you'll be using it opposite of what you normally use the pattern pieces for.
+That said, the pattern is available both ways. So you can print out the whole thing, tape all the pages together, and use that to cut out the fabric, if you prefer. But we recommend only printing the part that you need to cut out.
+
+You will need to print out the pockets and pocket facings in the traditional way.
+
+</Note>
+
+### Materials
+ - **Main fabric**
+   - Cut the **2 pants** parts.
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/cutting/en.md:16]: Does this mean:
```suggestion
   - Cut **2 pants** parts.
```

or are there two parts for the pants?

File: markdown/org/docs/patterns/waralee/instructions/en.md [hunk 7]
```diff
@@ -1,54 +1,73 @@
...
-##### If you do want pockets, make them first
-If you're going to make the pockets, make them before putting one piece of fabric on the other. It's easier to work with the parts still separated.
+#### Front Pockets
 
-</Note>
+Okay, the facing is used twice (per pocket). The pocket itself (part 3) is cut-on-fold.
+
+You attach one facing to the pocket after folding the sides under (for finishing), align with the slanted side (or add a small SA, since you will be needing to sew this to the other side of the pocket, and that seam will need to be finished. I use French seams for my pockets, so reducing bulk is good). This will be the 'inside' of the pocket.
+
+Iron-on some interfacing on the wrong side of the 'pants' fabric. about 3cm larger than the pocket opening (I use pinking shears to cut this, so it's less obvious from the right side that the interfacing is there).
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/instructions/en.md:24]:
```suggestion
Iron-on some interfacing on the wrong side of the 'pants' fabric, about 3cm larger than the pocket opening (I use pinking shears to cut this, so it's less obvious from the right side that the interfacing is there).
```

File: markdown/org/docs/patterns/waralee/instructions/en.md [hunk 8]
```diff
@@ -1,54 +1,73 @@
...
+Cut the pocket as usual, and push everything to the wrong side. Check to make sure you have the inside/outside and facing correctly. (I cheat a lot with this step by attaching that piece of facing after I do this step. If I get the inside/outside wrong, no harm done). Press this well. I press open both seams before folding them back onto themselves. Now take the longer part of the facing you just cut the opening through and fold it like a single welt. Stitch it in place by sewing through the seam closest to the crotch. Fold under the last part of it, and sew that to the pocket. Sew the little triangles on the ends of the pocket to the facing and pocket.
+Now bring the whole thing back to the outside, and sew a seam around it, about 5mm from the edge. Push it back inside, press and stitch 6mm from the edge to complete the French seam.
 
-## Step 2: Hem the sides and bottom
+As a last step, I sew the side closest to the side through all layers. This anchors the pocket to that side well.
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/instructions/en.md:32]: Is this referring to the side of the pocket closest to the side of the pants? I got a little lost with "side closest to the side".

File: markdown/org/docs/patterns/waralee/instructions/en.md [hunk 9]
```diff
@@ -1,54 +1,73 @@
...
+
+Make sure you sew the top of the pocket into the waist hem.
+
+#### Back Pockets
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/instructions/en.md:36]: Is this the bit that the fixme above is referring to? It feels strange to have a heading with no body. What do you think about moving the fixme to this section so it is clear why it is missing?

File: markdown/org/docs/patterns/waralee/instructions/en.md [hunk 10]
```diff
@@ -1,54 +1,73 @@
...
-Take a quarter of your waist measurement. Add that number to 40cm (17”). Add your seam allowance. That is the length of each string. Cut out four strips of fabric that are that long and 5cm (2”) wide.
+The strings are just long tubes of fabric.
 
-For example, if my waist circumference is 84.6cm: 84.6cm divided by 4 is 21.2cm. 21.2cm plus 40cm is 61.2cm. 61.2cm plus my seam allowance of 1cm is 62.2cm. I will cut four strips of fabric that are 62.2cm by 5cm.
+- Take a quarter of your waist measurement. Add that number to 40cm (17”). Add your seam allowance. This is the length of each string. 
+- Cut out four strips of fabric that are that long and 5cm (2”) wide.
 
 <Note>
 
+For example, if my waist circumference is 84.6cm: 84.6cm divided by 4 is 21.2cm. 21.2cm plus 40cm is 61.2cm. 61.2cm plus my seam allowance of 1cm is 62.2cm. I will cut four strips of fabric that are 62.2cm by 5cm.
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/instructions/en.md:69]: Did you mean to move this into the note? It seems unrelated to the main topic ("You can lengthen the front ties").

File: markdown/org/docs/patterns/waralee/instructions/en.md [hunk 11]
```diff
@@ -1,54 +1,73 @@
...
+</Fixme>
 
-##### If you do want pockets, make them first
-If you're going to make the pockets, make them before putting one piece of fabric on the other. It's easier to work with the parts still separated.
+#### Front Pockets
 
-</Note>
+Okay, the facing is used twice (per pocket). The pocket itself (part 3) is cut-on-fold.
+
+You attach one facing to the pocket after folding the sides under (for finishing), align with the slanted side (or add a small SA, since you will be needing to sew this to the other side of the pocket, and that seam will need to be finished. I use French seams for my pockets, so reducing bulk is good). This will be the 'inside' of the pocket.
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/instructions/en.md:22]:
```suggestion
You attach one facing to the pocket after folding the sides under (for finishing), aligned with the slanted side (or add a small SA, since you will be needing to sew this to the other side of the pocket, and that seam will need to be finished. I use French seams for my pockets, so reducing bulk is good). This will be the 'inside' of the pocket.
```

File: markdown/org/docs/patterns/waralee/instructions/en.md [hunk 12]
```diff
@@ -1,54 +1,73 @@
...
+
+Iron-on some interfacing on the wrong side of the 'pants' fabric. about 3cm larger than the pocket opening (I use pinking shears to cut this, so it's less obvious from the right side that the interfacing is there).
 
-If desired, sew the pockets.
+Lay the other facing part on the right side of the 'pants', with most of the fabric towards the crotch. You want about 1/4", or 6mm, sticking out on the side-seam side.
+Lay the 'inside' of the pocket (the side without the already attached facing) on the right side of the 'pants'. Add another 2cm or so for the seam and French finish as mentioned above. Stitch the two lines of stitching as you would on any welt pocket.
```
Comment (nicholasdower) [inline markdown/org/docs/patterns/waralee/instructions/en.md:27]: I can't figure out the "Add another 2cm or so" bit. Does this mean cut the pocket bigger than the pattern?

**Comment by nicholasdower** [comment]
@bobgeorgethe3rd looks great to me. I made a few small suggestions, mostly typos.

The hardest part for me was the "Front Pockets" section in Step 1 of the instruction. I haven't sewn pockets before and had trouble figuring out where the interfacing goes and where to sew. Maybe you would prefer to have someone who has sewn pockets review that bit. @joostdecock?

File: markdown/org/docs/patterns/waralee/cutting/en.md (hunk 6 above)
Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/waralee/cutting/en.md:16]: yea not sure why I added the "the" there

File: markdown/org/docs/patterns/waralee/cutting/en.md (hunk 5 above)
Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/waralee/cutting/en.md:24]: I think just the will bit

File: markdown/org/docs/patterns/waralee/instructions/en.md (hunk 12 above)
Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/waralee/instructions/en.md]: removed

**Comment by bobgeorgethe3rd** [comment]
I forgot to remove the pocket part properly my bad. It's removed now

File: markdown/org/docs/patterns/waralee/instructions/en.md (hunk 10 above)
Comment (nicholasdower) [reply markdown/org/docs/patterns/waralee/instructions/en.md:49]: Just reopening this as I think it is still relevant.

**Comment by nicholasdower** [comment]
Thanks for the update @bobgeorgethe3rd. I just now realized that the pockets bit was pulled straight from Discord. I didn't realize, sorry.

**Comment by nicholasdower** [comment]
One last thing. Just want to clarify that you intended to drop this bit:
image

File: markdown/org/docs/patterns/waralee/instructions/en.md (hunk 10 above)
Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/waralee/instructions/en.md:49]: I thought I had moved that ocrrect posiition damn it git

Comment (bobgeorgethe3rd) [reply markdown/org/docs/patterns/waralee/instructions/en.md:70]: Actually re-reading. Yes I did but it should be a separate tip so have done that. it is an example so it can the clutter the bullet points.

**Comment by bobgeorgethe3rd** [comment]
> One last thing. Just want to clarify that you intended to drop this bit: image

I have re-added this as you are right this needs to remain as the pocket info has not been updated yet.

**Comment by bobgeorgethe3rd** [comment]
> Thanks for the update @bobgeorgethe3rd. I just now realized that the pockets bit was pulled straight from Discord. I didn't realize, sorry.

That's ok. Originally it was suppose to be part of this update but I couldn't make heads or tails of it and Shark will get to it when she can so it was just easy to split the two and have it ready for shark.

**Comment by nicholasdower** [comment]
I just had another look and it is perfect. Thanks for the great change and thanks for bearing with me through all the questions.
